examples/qubit_characterizations_example.py

```diff
@@ -2,7 +2,26 @@
 import cirq
 
 
-def main():
+def main(minimum_cliffords=5, maximum_cliffords=20, cliffords_step=5):
+    """
+    Examples of how to run various qubit characterizations.  This example
```

Comment:
Not mentioned in message

Reply:
Added to the PR description and will add to commit message as well.